src/plone/restapi/tests/test_services_history.py

```diff
@@ -134,3 +137,63 @@ def test_no_sharing(self):
         url = self.doc.absolute_url() + '/@history/0'
         response = self.api_session.get(url)
         self.assertNotIn('sharing', response.json())
+
+
+class TestHistoryEndpointTranslatedMessages(unittest.TestCase):
+    layer = PLONE_RESTAPI_DX_FUNCTIONAL_TESTING
+
+    def setUp(self):
+        self.app = self.layer['app']
+        self.portal = self.layer['portal']
+        self.portal_url = self.portal.absolute_url()
+        setRoles(self.portal, TEST_USER_ID, ['Manager'])
+
+        set_available_languages()
```

Comment:
These three shouldn't be necessary any more since this is now done in the [`PloneRestApiDXLayer`](https://github.com/plone/plone.restapi/blob/aef7c2000e74ab2f2fb6715c6c59500d438238d6/src/plone/restapi/testing.py#L140-L142)